sites/all/modules/reol_base/reol_base.field.inc

```diff
@@ -101,7 +101,7 @@ function reol_base_field_formatter_view($entity_type, $entity, $field, $instance
           $pub_prod = publizon_load($entity->localId);
           // If not on publizon, do nothing.
           if (!$pub_prod) {
-            continue;
+            break;
```

Comment:
Semantic change. Do we really want to break out of the loop?

Reply:
PHP 7.4 complained about mis-used `continue` statement.... it's not the loop that break's but the switch statement (which is the only construct inside that loop) that break. So here break will have the samme effect as the continue before that php 7.4 did throw notices about.